### ISSUE TITLE
postfix_exporter_kumina  exporter

### ISSUE THREAD

**Issue by ashleykleynhans** [mesaguy/ansible-prometheus#24]
Thanks for the amazing job you're doing on this project, I find it extremely useful and invaluable.

This is more of a question than an actual issue.

I would like to use the postfix_exporter_kumina exporter for some mailservers, but I see its not mentioned in the documentation, and in a `wip` directory.  Does this mean it is currently a "work in progress".

Will it still work if I include it as:

```yml
prometheus_components:
  - node_exporter
  - postfix_exporter_kumina
```

**Comment by mesaguy** [comment]
No, sorry. This exporter is a work in progress. The current postfix_exporter release (0.2.0) fails to built on most distros. I will be tracking 'master' in the next major release as 'master' builds without issue. I will keep this issue open until support has been added. Probably a couple weeks.
